Source/Models/VirtusizeUserData.swift

```diff
@@ -0,0 +1,48 @@
+//
```

Comment:
As said above we don't need to care about those

Reply:
See the commit bc0929e